09LessonNine/in-class-project/css/style.css

```diff
@@ -0,0 +1,142 @@
+header{
+  width: 100%;
+  height: 400px;;
+  background-blend-mode: multiply;
+  background-color: #6e6e6e;
+  background-image: url("../img/1.jpg");
+  text-align: center;
+  padding-top: 200px;
+}
+
+h4{
+  font-family: 'Roboto Condensed', sans-serif;
+  color: white;
+  text-transform: uppercase;
+  margin-bottom: 15px;
+}
+
+
+h1{
+font-family: 'Roboto Condensed', sans-serif;
+color: white;
+text-transform: uppercase;
+font-size: 50px;
+margin-bottom:20px;
+}
+
+h3{
+  font-family: Times, serif;
+  color:white;
+  margin-bottom: 20px;
+  font-size: 20px;
+}
+
+.button{
+background-color: #91b300;
+border:none;
+width: 150px;
+height: 30px;
+color: white;
+text-transform: uppercase;
+padding-top: 8px;
+padding-bottom: 8px;
+padding-left: 10px;
+padding-right: 10px;
+}
+
+a{
+  text-decoration: none;
+  font-family:'Roboto Condensed', sans-serif;
+  font-size: 10px;
+}
+
+/*CSS for service.html starts here*/
+.clearfix{
+display: block;
+content: "";
+clear: both;
+}
+
+.landscaping{
+  background-image: url("../img/featured-image-5.jpg");
+  background-repeat: no-repeat;
+  background-blend-mode: multiply;
+  background-color: #6e6e6e;
+  background-size: cover;
+  width: 69%;
+  float: right;
+  clear: right;
+  height: 530px;
+}
+
+.growing{
+  background-image: url("../img/featured-image-3.jpg");
```

Comment:
hmm can you think of why your `background image` isn't filling its container element?